### DISCUSSION TITLE
Get a property from view or Model

### DISCUSSION THREAD

**Discussion by kossa** [spatie/laravel-settings#21] [Q&A]
Hello,

How to read a property ? currently I'm using DB::table('settings')->....

There is another way ?

**Comment by rubenvanassche** [comment]
Hi @kossa,

You should inject your settings class somewhere, for example:

```php
resolve(GeneralSettings::class)->site_name
```

Now you can get the `site_name` property on the `GeneralSettings` class we use in the readme.

**Comment by kossa** [comment]
That's fine, it's great if there is a helper for that :)
